src/components/MDXContainer.js

```diff
@@ -90,6 +91,7 @@ const defaultComponents = {
   ButtonLink: (props) => <MDXButton as={Link} {...props} />,
   ButtonGroup: MDXButtonGroup,
   DoNotTranslate: ({ children }) => <>{children}</>,
+  ContributorNote,
```

Comment:
i think we may want to declare it like this

```suggestion
  CONTRIBUTOR_NOTE: ContributorNote,
```